tool/utils.ts

```diff
@@ -208,6 +218,7 @@ async function checkForMusl(): Promise<void> {
  */
 export async function getJSApi(
   outPath: string,
+  os: OSKind,
   options?: {ref: string} | {path: string}
 ): Promise<void> {
```

Comment:
You shouldn't need to pass in an `os` parameter here. This only ever needs to know the *local* OS because it's just figuring out how best to create a local symlink, so it can just refer directly to `process.platform`.

Same for `buildEmbeddedProtocol()`.